Maple2.Server.Game/PacketHandlers/InteractObjectHandler.cs

```diff
@@ -40,15 +40,13 @@ private enum Command : byte {
     private void HandleEnd(GameSession session, IByteReader packet) {
         string entityId = packet.ReadString();
 
-        if (entityId.StartsWith("BillBoard") && session.Field?.TryGetAdBalloon(entityId, out FieldInteract? interact) == true && interact.React()) {
-            session.Send(PlayerHostPacket.AdBalloonWindow((interact.Object as InteractBillBoardObject)!));
-        } else if (session.Field?.TryGetInteract(entityId, out interact) == true && interact.React()) {
+        if (session.Field?.TryGetInteract(entityId, out FieldInteract? interact) == true && interact.React()) {
             switch (interact.Value.Type) {
                 case InteractType.Mesh:
                     session.Send(InteractObjectPacket.Interact(interact));
                     break;
                 case InteractType.Telescope:
-                    // TODO: this should only be sent if telescope not found yet?
+                    session.Send(InteractObjectPacket.Interact(interact));
```

Comment:
Did you verify this TODO comment?

Reply:
Yeah, this is always sent, regardless of new or old.